contracts/SL2RD.sol

```diff
@@ -132,38 +209,109 @@ contract SL2RD is
         );
     }
 
-    /// @notice Overrides ERC-721 safeTransferFrom function and ensures
-    /// only SHARE approved wallets can recieve the SL2RD slot NFT.
+    /// @notice Allows for an ERC-721 token to be transferred to a new address.
+    /// @dev Overrides the ERC721 version to add additional check that ensures
+    // the recipient address is a SHARE approved wallet hash.
     function safeTransferFrom(
         address from_,
         address to_,
         uint256 tokenId_
     ) public override {
+        // Restrict trading of community slots until distribution process is complete.
+        if (_nextAvailableCommunitySlot < _totalCommunitySlots) {
```

Comment:
Nice! Transfers are restricted to the community partition until community distribution is complete, at which point both partitions are open to transfers.

Reply:
Yup, this became clear through the dialogue in the doc.